oximeter/db/src/oxql/query/mod.rs

```diff
@@ -362,6 +366,32 @@ impl Query {
     pub(crate) fn parsed_query(&self) -> &QueryNode {
         &self.parsed
     }
+
+    pub(crate) fn add_filters(&self, filters: Vec<(String, Uuid)>) -> Self {
```

Comment:
`Vec<(String, Uuid)>` is an appalling type for this. I almost want to construct the entire vec of filters outside this function and pass it in.